policy/modules/system/selinuxutil.te

```diff
@@ -223,6 +223,10 @@ optional_policy(`
         unconfined_dontaudit_read_pipes(load_policy_t)
 ')
 
+optional_policy(`
+	portage_use_inherited_ptys(load_policy_t)
```

Comment:
It would seem that the ptys are not normally inherited, as fds are not inherited (see line 218) so this should probably be a dontaudit instead. It should also be in the above portage block too.